src/main/java/org/fcrepo/auth/webac/WebACRolesProvider.java

```diff
@@ -138,7 +147,7 @@ public void deleteRoles(final Node node) throws RepositoryException {
         // Read the effective Acl and return a list of acl:Authorization statements
         final List<WebACAuthorization> authorizations = effectiveAcl
                 .map(uncheck(x -> getAuthorizations(x.getLeft().toString())))
-                .orElse(new ArrayList<>());
+                .orElseGet(() -> getDefaultAuthorizations());
```

Comment:
Instead of tying in the default ACL here, I would have thought the backstop ACL would logically be returned when searching for `getEffectiveAcl`:
https://github.com/acoburn/fcrepo-module-auth-webac/blob/fcrepo-1792/src/main/java/org/fcrepo/auth/webac/WebACRolesProvider.java#L121

Was there a reason you did not go that direction?